modules/data.atmosphere/R/met.process.R

```diff
@@ -55,35 +55,32 @@ met.process <- function(site, input_met, start_date, end_date, model, host, dbpa
           
           args <- list(outfolder, start_date, end_date)
           cmdFcn  = paste0(pkg,"::",fcn,"(",paste0("'",args,"'",collapse=","),")")
-          remote.execute.R(cmdFcn,host$name,user=NA, verbose=TRUE)
-          
-          mimetype = 'application/x-netcdf'
-          formatname = "NARR"
-          check <- dbfile.input.check(site.id, start_date, end_date, mimetype, formatname, con=con, hostname=fqdn())
+          new.files <- remote.execute.R(cmdFcn,host$name,user=NA, verbose=TRUE)
           
+          print("start CHECK")
+          check = dbfile.input.check(site$id, start_date, end_date, 
+                                     mimetype=new.files$mimetype[1], formatname=new.files$formatname[1], 
+                                     con=con, hostname=new.files$host[1])
+          print("end CHECK")
+          print(check)
           if(length(check)>0){
-            raw.id = check$container_id[1]
+            raw.id <- list(input.id=check$container_id, dbfile.id=check$id)
           }else{
-            raw.id <- dbfile.input.insert(in.path = outfolder, 
-                                          in.prefix = "NARR", 
-                                          siteid = 1135, 
+            raw.id <- dbfile.input.insert(in.path=dirname(new.files$file[1]),
```

Comment:
yes right now the workflow wouldn't know how to look up each met file individually. Instead we are storing the containing folder and the prefix - which in the case of ameriflux is the site code.